src/wasm-binary.h

```diff
@@ -261,17 +261,32 @@ enum ASTNodes {
   F64Le = 0x9a,
   F64Gt = 0x9b,
   F64Ge = 0x9c,
-  I32SConvertF32 = 0x9d,
-  I32SConvertF64 = 0x9e,
-  I32UConvertF32 = 0x9f,
-  I32UConvertF64 = 0xa0,
-  I32ConvertI64 = 0xa1,
```

Comment:
The v8 source calls it `I64SConvertI32` for example, but it maps cleaner to the .wast format (i64.convert_s/i32) if you call it `I64ConvertSI32`